core/src/rpc_service.rs

```diff
@@ -20,6 +20,17 @@ use std::{
 };
 use tokio::prelude::Future;
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
```

Comment:
I think we’ll want these structs in https://github.com/solana-labs/solana/blob/master/client/src/rpc_request.rs so they’re available to the client-side rust code.

Reply:
ah sorry! misread that as `rpc_service`, thank you!

Reply:
(added)